components/FAQ/FAQ.tsx

```diff
@@ -0,0 +1,38 @@
+'use client'
+
+import { Montserrat } from "next/font/google";
+import LinkButton from "@/LinkButton"
+import React, { useState } from 'react'
+import FAQAccordion from "./FAQAccordion"
+import data from "./FAQ.json"
+
+const montserrat = Montserrat({ subsets: ['cyrillic'], weight: ['400', '500', '700'] });
+
+
+export default function FAQ() {
+  const [activeId, setActiveId] = useState(null)
+
+  return (
+    <div className="grid lg:grid-cols-2 gap-10 p-10 w-4/5 m-auto sm:grid-cols-1">
+      <div className="flex flex-col items-start gap-5">
+        <h2 className='text-5xl font-bold'>Често поставувани
+          прашања</h2>
+        <p className={`'font-medium text-xl' ${montserrat.className}`}>Погледнете дали вашето прашање се наоѓа во овие прашања и договори. Доколку не се наоѓа, слобдно контактирајте не преку копчето подолу.</p>
+        <LinkButton type={'link'} url={''} text={'Поставете прашање'} bgColor={'bg-orange'} textColor={'text-black'}></LinkButton>
+      </div>
+      <div className="flex flex-col justify-between items-start gap-5 size-full">
+        {data.faq.map(({ id, question, answer }) =>
+          <div className="flex grow size-full items-center justify-between" onClick={() => {
```

Comment:
You shouldn't do inline callbacks. It makes the code more difficult to read. Instead make a callback before `return` and then use it here.

https://stackoverflow.com/questions/73615347/passing-a-parameter-to-onclick-function-nextjs